package.json

```diff
@@ -12,15 +12,32 @@
     "url": "https://github.com/eventualbuddha/keysim.js"
   },
   "scripts": {
-    "build": "gobble build dist -f",
+    "prebuild": "mkdir -p dist",
+    "build:node": "npm run prebuild && babel lib/keysim.es > dist/keysim.node.js",
+    "build:browser": "npm run prebuild && browserify . --standalone keysim > dist/keysim.browser.js",
```

Comment:
So, it uses the original source. It then uses babelify to transpile it on the fly, using the same config that is used in the `build:node` script.